### ISSUE TITLE
Default region & zone?

### ISSUE THREAD

**Issue by max-sixty** [GoogleCloudPlatform/k8s-config-connector#250]
Would it be possible to set these fields from config-connector?

https://cloud.google.com/compute/docs/regions-zones/changing-default-zone-region#gcloud

**Comment by spew** [comment]
Hi @max-sixty thank you for the feature request. At what context would you like to be able to set this? At the namespace level?

**Comment by max-sixty** [comment]
Maybe I'm thinking about this badly — I was thinking there would be some resource like `project`, and it would have properties like `default-zone`?

**Comment by spew** [comment]
Yes, I see what you are saying. What if on the namespace there was a "context". In the context there would be a Project -- (the project that resources are assigned to by default) and other properties like region & zone ?

**Comment by max-sixty** [comment]
That sounds ideal, thanks @spew
